cmd/kubeadm/app/util/arguments.go

```diff
@@ -22,6 +22,7 @@ import (
 	"strings"
 
 	"github.com/pkg/errors"
+	"k8s.io/klog/v2"
```

Comment:
should be one line bellow (to have pkg/errors in a separate import group), but other than that, LGTM.
thanks for the PR.

Reply:
Updated!